.gitlab-ci.yml

```diff
@@ -740,13 +742,42 @@ trigger-simnet:
     - if: $CI_COMMIT_REF_NAME == "master"
   needs:
     - job:                         publish-docker-substrate
-  # `build.env` brings here `$IMAGE_NAME` and `$IMAGE_TAG` (`$VERSION` here,
-  # i.e. `2643-0.8.29-5f689e0a-6b24dc54`).
-  variables:
-    TRGR_PROJECT:                  ${CI_PROJECT_NAME}
-    TRGR_REF:                      ${CI_COMMIT_REF_NAME}
-    # Simnet project ID
-    DWNSTRM_ID:                    332
-  script:
-    # API trigger for a Simnet job
-    - .maintain/gitlab/trigger_pipeline.sh --simnet-version=${SIMNET_REF}
+  # variables:
+  # `build.env` brings here `${SUBSTRATE_IMAGE_NAME}` and `${SUBSTRATE_IMAGE_TAG}`
+  # (`$VERSION` here, # i.e. `2643-0.8.29-5f689e0a-6b24dc54`).
+  # ${SIMNET_REF} is a gitlab variable
```

Comment:
you could also print out the variables here as you did in polkadot

Reply:
will do